### DISCUSSION TITLE
How to Develop overlapping multi labeled segmentation.

### DISCUSSION THREAD

**Discussion by erenkal** [Project-MONAI/MONAI#4267] [Q&A]
I have 20 labels some of them are overlapping. Data in 4D nifti format [x][y][z][ labels ]   labels are 1 and 0 in a list.
How should I use the monai features ? 
Should I train a model with 20 input channels ?

**Comment by wyli** [comment]
yes, you can save it into shape [x, y, z, 20], along the last dimension are binary vectors, and use a 20-channel model. just in the segmentation loss/metric functions set `to_onehot_y=False`, and `softmax=False`, `sigmoid=True` (to avoid the assumption of mutually exclusive labels).

**Comment by erenkal** [comment]
How should I create dataset and dataloader ?
Are there any examples?
